src/app/additional/additional.component.html

```diff
@@ -111,7 +111,7 @@ <h2 class="warranty-wiz--subheader">Let's dig a little deeper.</h2>
         Previous
       </button>
       <button id="additional-submit--btn" name="button" class="step__footer__continue-btn btn" style="float:right;"
-              type="submit">
+              type="submit" ng-disabled="imageResizing" [ngStyle]="{'background-color': imageResizerService.disabledButtonColor(imageResizing) }">
```

Comment:
this is an interesting one. good find.

Reply:
Ok I see why you made the service public.  But, I'm going to throw a flag on it, the service and presentation layers shouldn't know anything about each other, and this goes both ways, a service should never "know" anything about ui be it styles, or state.